src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.util.Map;
+import service.FileReaderService;
+import service.FileWriterService;
+import service.ReportService;
+import service.impl.FileReaderServiceImpl;
+import service.impl.ReportServiceImpl;
+import service.impl.ReportWriterService;
+
+public class Main {
+    private static final String PATH_TO_FILE = "src/main/java/resources/File.csv";
+
+    public static void main(String[] args) {
+        FileReaderService fileReaderService = new FileReaderServiceImpl();
+        ReportService reportService = new ReportServiceImpl();
+        FileWriterService fileWriterService = new ReportWriterService();
+        Map<String, Integer> report = reportService
+                .createReport(fileReaderService
+                        .readFromFile(new File(PATH_TO_FILE)));
+        fileWriterService.writeToFile(report);
```

Comment:
at the moment, main method not looks like at this task diagram:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/de55f492-bce3-4e5e-9da8-815e0120097c)

thats why:
1. cant see at this method result of converting data from file into FruitTransactions
2. you process your data right at the `reportService`, not in the separate class using list of FruitTransactions
3. report creating should not accept any parameters, it should read directly from DB

Reply:
Done